src/components/Header/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Hamburger from './hamburger';
+
+import './header.css';
+
+const Header = ({onHamburgerClick, isHamburgerActive}) => (
```

Comment:
Why you export just one item for module use `export default` option.
`export default Header`